### PR TITLE
Task was destroyed but is pending!

### PR DESCRIPTION
**Pull request by al-little**
Added a set to keep a reference to the tasks while in progress so that they are not garbage collected.

### CODE REVIEW

**Comment by y-martini** [comment]
@al-little what about tests?

**Comment by al-little** [comment]
Hi Yuriy,

The bug occurs when longer running tasks get garbage collected before
completing.
I am using asyncio and the Bleak Bluetooth library and was getting the
exception occasionally. I am relatively new to Python development and found
it difficult to create a failing test.

Thanks,

Al

File: events_manager/__init__.py [hunk 1]
```diff
@@ -82,11 +84,13 @@ def get_listeners(
     return _listeners.get(event_type, [])
 
 
-async def _run_listeners(e: Event) -> None:
+async def __run_listeners(e: Event) -> None:
```
Comment (albertogferrario) [inline events_manager/__init__.py]: Could you please explain why you consider to alter its visibility?

**Comment by albertogferrario** [comment]
Hello @al-little, thanks for being a `fucktory` contributor. 
I reviewed the contribution and left some comments. I will be happy to answer any questions.

**Review by albertogferrario — CHANGES_REQUESTED** [review]
The following changes have to be provided before merge:
- [ ] remove blank lines at 75 and 77;
- [ ] restore logging and blank line at line 74 and 75.

File: events_manager/__init__.py (hunk 1 above)
Comment (al-little) [reply events_manager/__init__.py]: I accidentally added dunder '__'. The original code used single '_'.

**Comment by al-little** [comment]
Morning. Any chance of progressing the pull request? Please let me know if there is anything else I need to do.

**Review by albertogferrario — APPROVED** [review]
Thank you for your contribution!

**Comment by albertogferrario** [comment]
Hello @al-little, sorry for bothering you but checks fail because of lint issues, you can find more details [here](https://github.com/webfucktory/python-events-manager/actions/runs/5264436242/jobs/9540667164).
I suggest you to run the `lint` script locally, you can find it in repository root.
If you have any doubts do not hesitate to contact me.
Thank you.

**Pull request merged**